controllers/business_profile_controller.js

```diff
@@ -2,6 +2,22 @@
 
 module.exports = (connection) => {
   return (req, res) => {
-    res.render('pages/business_profile');
+    let user_id = req.params.id;
```

Comment:
Is the _/personal/:id_ page only meant for logged in users? If that's the case it might be preferred to access the `user_id` from the `req.session.user` object: https://github.com/JiejayLan/CSC336_group_project/blob/c877b695f173b306d0c901fb9af48963bcb46b1b/server.js#L19-L36

Reply:
It is not only meant for logged in users, since we have to allow employee to access the employer profile and vas versa. Therefore, the design should be when ever you click into either one of the site. You are require to input a id to identify the user you want to display.

Reply:
Got it. I figured that might've been the case.